winui/Masked-TextBox/Overview.md

```diff
@@ -19,4 +19,5 @@ The [WinUI MaskedTextBox](https://www.syncfusion.com/winui-controls/masked-textb
 * `PromptChar`: Customize the prompt characters used to set the default characters as input in the absence of input.
 * `Value`: Enter values and clipboard operations in the control based on the ValueMaskFormat property.
 * `ValueMaskFormat`: Setting formatted values in the control using mask format values such as prompt, literals, and both.
-* `Customization`: Customize the UI of the masked text box.
\ No newline at end of file
+* `Customization`: Customize the UI of the masked text box.
```

Comment:
@GilbertmanojSF3971 Remove Customization

Indicates errors or other information by displaying an icon and also show additional details when hovering the cursor over the icon.

Reply:
Hi @Manivannan-E Sir. Customization title is already present in the control. And i have modified the Content for the Error Indication title.